<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelProperty_UI</class>
 <widget class="QWidget" name="ModelProperty_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型属性</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>模型名称</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="modelName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;新模型&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="handIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>空间信息</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="pos_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>缩放</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_y">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>位置</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_z">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotate_y">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scale_y">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotate_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>旋转</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotate_z">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scale_z">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scale_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>属性信息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mesh名称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modelLib">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="posSelcted">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑模型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pos_x</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotPosXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos_y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotPosYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pos_z</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotPosZChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_x</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotRoateXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotRoateYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotate_z</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotRoateZChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_x</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotScaleXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_y</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotScaleYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_z</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotScaleZChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>handIn</sender>
   <signal>clicked()</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotHandIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posSelcted</sender>
   <signal>clicked()</signal>
   <receiver>ModelProperty_UI</receiver>
   <slot>slotPosSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotPosXChanged(double)</slot>
  <slot>slotPosYChanged(double)</slot>
  <slot>slotPosZChanged(double)</slot>
  <slot>slotRoateXChanged(double)</slot>
  <slot>slotRoateYChanged(double)</slot>
  <slot>slotRoateZChanged(double)</slot>
  <slot>slotScaleXChanged(double)</slot>
  <slot>slotScaleYChanged(double)</slot>
  <slot>slotScaleZChanged(double)</slot>
  <slot>slotHandIn()</slot>
  <slot>slotCancel()</slot>
  <slot>slotPosSelected()</slot>
 </slots>
</ui>
